internal/commands/build_test.go

```diff
@@ -38,21 +38,23 @@ func TestBuildCommand(t *testing.T) {
 
 func testBuildCommand(t *testing.T, when spec.G, it spec.S) {
 	var (
-		command        *cobra.Command
-		logger         *logging.LogWithWriters
-		outBuf         bytes.Buffer
-		mockController *gomock.Controller
-		mockClient     *testmocks.MockPackClient
-		cfg            config.Config
+		command                *cobra.Command
+		logger                 *logging.LogWithWriters
+		outBuf                 bytes.Buffer
+		mockController         *gomock.Controller
+		mockClient             *testmocks.MockPackClient
+		cfg                    config.Config
+		imagePullPolicyHandler image.ImagePullPolicyHandler
 	)
 
 	it.Before(func() {
 		logger = logging.NewLogWithWriters(&outBuf, &outBuf)
 		cfg = config.Config{}
 		mockController = gomock.NewController(t)
 		mockClient = testmocks.NewMockPackClient(mockController)
+		imagePullPolicyHandler = image.NewPullPolicyManager(logger)
```

Comment:
@jjbustamante  currently we are spawning `imagePullPolicyHandler` for every test cases. Is it ok? or do we need some kind of mock here ?

Reply:
I think we can create a mock, at this point we don't want to use the real component because we want to test only the build logic.